WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/builders/ScanStateListItemsBuilder.kt

```diff
@@ -242,16 +246,28 @@ class ScanStateListItemsBuilder @Inject constructor(
         )
     }
 
-    private fun buildThreatsFoundDescription(site: SiteModel, threatsCount: Int) = DescriptionState(
-        UiStringText(
-            htmlMessageUtils
-                .getHtmlMessageFromStringFormatResId(
-                    R.string.scan_idle_threats_found_description,
-                    "<b>$threatsCount</b>",
-                    "<b>${site.name ?: resourceProvider.getString(R.string.scan_this_site)}</b>"
-                )
+    private fun buildThreatsFoundDescription(
+        site: SiteModel,
+        threatsCount: Int,
+        onHelpClicked: () -> Unit
+    ): DescriptionState {
+        val descriptionText = htmlMessageUtils
+            .getHtmlMessageFromStringFormatResId(
+                R.string.scan_idle_threats_found_description,
+                "<b>$threatsCount</b>",
+                "<b>${site.name ?: resourceProvider.getString(R.string.scan_this_site)}</b>"
+            )
+
+        val clickableText = resourceProvider.getString(R.string.scan_here_to_help)
+        val startIndex = descriptionText.indexOf(clickableText)
```

Comment:
⚠️ I'm worried that this might not work as expected. No-one can guarantee that translations to other languages will have the exact same translation for `scan_here_to_help` as what is used in `scan_idle_threats_found_description` -> so I think this might return -1. Or am I missing something?


What if we used a string argument instead of relying on the translation. Wdyt?

```
<string name="scan_idle_threats_found_description">The scan found %1$s potential threats on %2$s. Please review them below and take action. We\'re %3$s if you need us.</string>
<string name="scan_here_to_help">here to help</string>
```

Wdyt?

Reply:
Oops, yes you're right....will fix it shortly.

Reply:
Fixed in 6dbdce9.